src/layout/DwindleLayout.cpp

```diff
@@ -521,6 +527,63 @@ void CHyprDwindleLayout::resizeActiveWindow(const Vector2D& pixResize, CWindow*
     const bool DISPLAYTOP    = STICKS(PWINDOW->m_vPosition.y, PMONITOR->vecPosition.y + PMONITOR->vecReservedTopLeft.y);
     const bool DISPLAYBOTTOM = STICKS(PWINDOW->m_vPosition.y + PWINDOW->m_vSize.y, PMONITOR->vecPosition.y + PMONITOR->vecSize.y - PMONITOR->vecReservedBottomRight.y);
 
+    if (PWINDOW->m_bIsPseudotiled) {
+        if (!m_PseudoDragFlags.started) {
+            m_PseudoDragFlags.started = true;
+
+            const auto PBORDERSIZE = &g_pConfigManager->getConfigValuePtr("general:border_size")->intValue;
```

Comment:
1. missing static
2. any particular reason not to use recalcSizePosRecursive?

Reply:
1. I didn't use it because you didn't seem to use it yourself in most places. (e.g. `applyNodeDataToWindow`)
2. I thought about it for a minute and decided to use `m_vRealSize` since it should already be at the right size before you start dragging it.

Reply:
1. did I miss any? cock shit piss fuck